packages/app/src/commands/modernize.js

```diff
@@ -0,0 +1,162 @@
+/**
+ * Example usage:
+ *
+ * d2-utils modernize \
```

Comment:
This would be great as a `jsdoc` comment to auto-generate documentation!  For now fine to keep it as a comment here :-)

(side-note: should be `d2-app i18n modernize` now)